### PR TITLE
Added 'input-history' feature in CLI

### PR DESCRIPTION
**Pull request by mayankDhiman**
1. Created a proper promt for VisMa, handeling all kinds of user interactions.
2. This adds input-history feature to CLI.
3. Refactored code a bit to make UI more user friendly.
addresses #62 
sample: https://pastebin.com/k2F3WZnG

### CODE REVIEW

**Comment by mayankDhiman** [comment]
Hi @8hantanu & @sidkothiyal, 
I have changed arguments of `default` method of `cmd` library. Thats the reason why Codacy is showing an error. But, for our implementation modifying `default` and keeping all other methods like `simplify`, `integrate` etc. wrapped inside it is necessary and this would indeed promote cleaner code (if we write all such methods like simplify etc. there would be code repetition and reduced readability). Thus, I think we can ignore this Codacy warning. What do you say? :)

File: main.py [hunk 1]
```diff
@@ -1,46 +1,71 @@
...
-    while cin != 'exit':
-        if cin == 'gui':
+
+    class VisMa_Prompt(Cmd):
+        '''This inititates the main VisMa Prompt from where user may move to CLI/GUI'''
+
+        userManual = "|_________________________________________________________________________________________________|\n"\
+                     "| gui  ->> opens Visma in GUI mode.                                                               |\n"\
+                     "| exit ->> Closes the prompt.                                                                     |\n"\
+                     "| exit ->> Closes the prompt.                                                                     |\n"\
```
Comment (8hantanu) [inline main.py:20]: Line repeated.
```suggestion
```

**Review by 8hantanu — COMMENTED** [review]
> I have changed arguments of ...

That's not a problem. You can ignore certain codacy warnings. As long as pytest tests are passing its alright.

In the CLI when giving no command the latest output is shown again.
```
>>> help

Documented commands (type help <topic>):
========================================
EOF  exit  gui  help  manual

>>>

Documented commands (type help <topic>):
========================================
EOF  exit  gui  help  manual
```

File: main.py (hunk 1 above)
Comment (mayankDhiman) [reply main.py:20]: oops changed that. XD

**Comment by mayankDhiman** [comment]
new sample output:
https://pastebin.com/eX37gLwD

**Pull request merged**